content/en/watchdog/rca.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Watchdog RCA
+kind: documentation
+further_reading:
+- link: "/watchdog/faq/root-cause-not-showing/"
+  tag: "Documentation"
+  text: "Root cause not showing"
+---
```

Comment:
What about including this blog? https://www.datadoghq.com/blog/datadog-watchdog-automated-root-cause-analysis/

Reply:
this blog post will be replaced soon